### PR TITLE
Add Toggle 

### PR DESCRIPTION
**Pull request by Blaine-Mason**
Implement a toggle switch that activates a pending listing.  Connect route and ensure listing appears in the listing view as "active".

### CODE REVIEW

File: templates/admin/admin_edit_listing.html [hunk 1]
```diff
@@ -4,6 +4,12 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style>
```
Comment (jacobduncan00) [inline templates/admin/admin_edit_listing.html]: Although this works, it's standard to keep our styling conventions in the Vue component it is being used

**Comment by jacobduncan00** [comment]
Besides moving the style tag to the Vue component, can you also implement this for the active listings as well so we can de-activate those if needed

**Comment by jacobduncan00** [comment]
> Besides moving the style tag to the Vue component, can you also implement this for the active listings as well so we can de-activate those if needed

Disregard, I did this in my PR

**Comment by Blaine-Mason** [comment]
So is the toggle on the pending still okay?

**Comment by jacobduncan00** [comment]
> So is the toggle on the pending still okay?

Yes, that is good. Just move the style tag to the component and gtg

**Comment by Blaine-Mason** [comment]
on it!

**Comment by jacobduncan00** [comment]
I implemented the toggle in my branch, it's essentially your code but got put into my branch because I had some stuff going in that was related to the /edit-listing, so you can by all means take credit for the activate-deactive toggle and will note that this PR got pulled into my [Admin Edit Listing Revamp PR](https://github.com/jventura1738/InternshipPortal/pull/27)

**Pull request closed without merging**